### PR TITLE
Sroberge/2.0.1

### PR DESCRIPTION
**Pull request by vim-sroberge**

### CODE REVIEW

**Comment by vim-sroberge** [comment]
All fiddles are updated to this version.

File: src/vim-webgl-viewer/viewerSettings.ts [hunk 1]
```diff
@@ -28,33 +28,68 @@ export type Settings = {
...
+
+    /**
+     * Camera zoom level
+     * Default: 1
+     */
     zoom: number
-    /** Initial forward of the camera */
+
+    /**
+     * Initial forward of the camera
```
Comment (mavimaec) [inline src/vim-webgl-viewer/viewerSettings.ts:90]:
```suggestion
     * Initial forward vector of the camera
```

File: src/vim-webgl-viewer/viewerSettings.ts [hunk 2]
```diff
@@ -64,111 +99,224 @@ export type Settings = {
...
-      /** Selection outline intensity. */
-      intensity: number;
-      /** Selection outline falloff. */
-      falloff: number;
-      /** Selection outline blur. */
-      blur: number;
-      /** Selecetion outline color. */
-      color: THREE.Color;
-    }
+  * Isolation materials options
```
Comment (mavimaec) [inline src/vim-webgl-viewer/viewerSettings.ts:229]:
```suggestion
  * Isolation material options
```

File: src/vim-webgl-viewer/viewerSettings.ts [hunk 3]
```diff
@@ -64,111 +99,224 @@ export type Settings = {
...
+  */
+  isolation: {
+    /**
+    * Isolation materials color
```
Comment (mavimaec) [inline src/vim-webgl-viewer/viewerSettings.ts:233]:
```suggestion
    * Isolation material color
```

File: src/vim-webgl-viewer/viewerSettings.ts [hunk 4]
```diff
@@ -64,111 +99,224 @@ export type Settings = {
...
+    * Default: rgb(78, 82, 92)
+    */
+    color: THREE.Color
+    /**
+    * Isolation materials opacity
```
Comment (mavimaec) [inline src/vim-webgl-viewer/viewerSettings.ts:238]:
```suggestion
    * Isolation material opacity
```

**Review by mavimaec — APPROVED** [review]
Thanks @vim-sroberge !

**Pull request merged**